tests/rptest/clients/kafka_cli_tools.py

```diff
@@ -183,6 +183,15 @@ def produce(self,
         ]
         self._execute(cmd)
 
+    def consume(self, topic, num_records, fetch_size=1048576):
+        self._redpanda.logger.debug("Consuming from topic: %s", topic)
+        cmd = [self._script("kafka-consumer-perf-test.sh")]
+        cmd += ["--topic", topic]
+        cmd += ["--messages", str(num_records)]
+        cmd += ["--fetch-size", str(fetch_size)]
+        cmd += ["--broker-list", self._redpanda.brokers()]
+        return self._execute(cmd)
```

Comment:
If we want to wrap the perf-test script then I think we should rename the interface something other than `consume`, which with such a name I think has a reasonable expectation to consume data and return it. But this method does something else. If go with this, it would also be useful to return some of the metadata already pre-parsed so the caller doesn't have do that.

Another option for this PR is to remove this `consume` wrapper, and fetch the data using a client directly in your test and count the resulting bytes. You could wrap something like `kafka-console-consumer.sh` but it would probably be far far easier to use the native python kafka client. I think that is available as `from kafka import KafkaConsumer`. There you can read the bytes directly from teh topic into your test.